data retrieval/src/main/java/de/tinf15b4/kino/retrieval/scraper/KinemathekScraper.java

```diff
@@ -75,7 +75,7 @@ private void handleMovies(List<WebElement> movieElements, LocalDate date) {
             Playlist playlist = new Playlist();
             playlist.setCinema(cinema);
             playlist.setMovie(movie);
-            playlist.setTime(Date.from(dateTime.atZone(ZoneId.of("GMT+1")).toInstant()));
+            playlist.setTime(Date.from(dateTime.atZone(ZoneId.of("GMT+2")).toInstant()));
```

Comment:
I actually didnt know there is something like that. I am changing that immediatelly ^^